static/404.html

```diff
@@ -1,20 +1,31 @@
+
 <!DOCTYPE html>
+
+
 <html lang="en">
-  <head>
-    <meta charset="utf-8">
-    <script>sessionStorage.redirect = location.href</script>
-    <meta http-equiv="refresh" content="0;URL='/'"></meta>
-  </head>
-  <body>
-    <!--
-    This file will be returned by github-pages servers when the user visits
+
```

Comment:
Does prettier add all this whitespace? 😕

Reply:
I don't think we should run html with prettier. Let me revise.

Reply:
prettier now ignores HTML files. Let me know if we need additional changes before this can get merged.